### ISSUE TITLE
Add more colors

### ISSUE THREAD

**Issue by jdiaz97** [psiphi75/sonogram#17]
I think it would be cool to have these palettes: https://www.thinkingondata.com/something-about-viridis-library/
I can do a PR, but I'd like to know if this package is still mantianed

**Comment by psiphi75** [comment]
Yes, create the PR and I'll review and merge it, then publish the package to crates.io. 
Those colour palettes look cool.
